src/components/layout/NotificationSnackBar/DonationNotificationLayout.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslation } from 'next-i18next'
+import { Avatar, Box, Typography } from '@mui/material'
+import { dateToTime } from 'components/faq/helpers/utils'
+import { grey } from '@mui/material/colors'
+import { money } from 'common/util/money'
+
+export type NotificationLayoutData = {
+  createdAt: string
+  person: {
+    firstName: string
+    lastName: string
+    picture: string
+  }
+  currency: string
+  amount: number
+}
+
+export default function DonationNotificationLayout({
+  data,
+}: {
+  data: NotificationLayoutData | undefined
+}) {
+  const { person, createdAt, currency, amount } = data || {}
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center', maxHeight: '50px' }}>
+      <Avatar sx={{ marginRight: '1rem' }} src={person?.picture} alt="Avatar" />
+      <Box>
+        <Typography component="h1" sx={{ fontWeight: 'bold' }}>
+          {person ? `${person?.firstName} ${person?.lastName}` : 'unknown'}
+        </Typography>
```

Comment:
Should have some better default than "unkown" here. Probablye "Anonymous" would be most fitting

Reply:
I don't knpw

Reply:
Thinking about it Anonymous sounds better I will change it :)